en/docs/services/create-your-first-service.md

```diff
@@ -153,10 +153,12 @@ To deploy the `covid-stats` service, follow this procedure:
 
     ![Deployment Logs](../assets/img/services/deployment-logs.png){.cInlineImage-full}
 
-3. Once the service is deployed, click the **Go Live** tab below the low-code diagram. Then click **://cURL** and copy the cURL command.
+3. Once the service is deployed, click on **://cURL**.
 
     ![Get cURL command](../assets/img/services/copy-curl-command.png){.cInlineImage-full}
```

Comment:
There is not curl command here. Is this the correct Image?

Reply:
Yes. The curl command shows in the next dialog box that opens. The image just shows the **://cURL** button. The text that directs the user to copy the curl command comes after the image.